discovery-service/src/main/resources/discovery-service-log-messages.yml

```diff
@@ -0,0 +1,22 @@
+messages:
+
+    - key: apiml.discovery.http
+      number: ZWEAD100
+      type: WARNING
+      text: "Discovery service is configured to use insecure an HTTP protocol"
+
+    - key: apiml.discovery.staticDefinitionsDirectoryNotValid
+      number: ZWEAD101
+      type: WARNING
+      text: "Static Api definition directory '%s' is not a directory or does not exist"
+
+    - key: apiml.discovery.errorParsingStaticDefinitionFile
+      number: ZWEAD102
+      type: ERROR
+      text: "Error loading the static Api definition file '%s'"
+
+    - key: apiml.discovery.errorParsingStaticDefinitionData
+      number: ZWEAD103
+      type: WARNING
+      text: "Unable to process static Api definition data: '%s'"
+
```

Comment:
How about to create a message keys in one style? I mean someone create them with first letter of last part in uppercase (example: com.ca.mfaas.product.common.BuildInfoPropertiesNotFound), someone in lowercase (example: apiml.discovery.errorParsingStaticDefinitionData).

Reply:
Good catch, thanks!